commands/mod.js

```diff
@@ -12,18 +12,18 @@ module.exports = {
     .setDescription('Moderation commands')
     .addSubcommand(
       new SlashCommandSubcommandBuilder()
-        .setName("mute")
-        .setDescription("Mutes a user via your configured muted role.")
+        .setName("timeout")
+        .setDescription("Timeouts a user via your configured timeoutd role.")
```

Comment:
This is inaccurate. xd. This does not assign any roles. The old description was also inaccurate. May want to update to something like `Times out designated user for x amount of time.`